shared/js/background/redirect.es6.js

```diff
@@ -120,6 +121,14 @@ function handleRequest (requestData) {
         }
 
         if (tracker) {
+            // temp allowlisted trackers to fix site breakage
+            const allowListed = trackerAllowlist(thisTab.site.url, requestData.url)
+            if (allowListed && utils.isFeatureEnabled('trackerAllowlist')) {
```

Comment:
1. @jonathanKingston wrote a new helper to get this info from the `site` object.
 2. Can we check if the feature is enabled for the site before doing matching?